src/io/pti.jl

```diff
@@ -943,4 +943,744 @@ function _populate_defaults!(data::Dict)
+
+        else
+            # Get default owner
+            bus_i = branch["f_bus"]
+            owner = bus_owner[bus_i]
+            
+            # Get Dict in a PSSE way
+            psse_comp = _pm2psse_branch(branch, owner)
+            
+            # Print it in the file
+            _print_pti_str(io, psse_comp, _pti_dtypes["BRANCH"])
+        end
+    end
+    
+    println(io, "0 / END OF BRANCH DATA, BEGIN TRANSFORMER DATA")
+    
+    # Transformers
+    for (type, transformer) in transformers
+        # Get default transformer base
+        sbase = data["baseMVA"]
+
+        if type == "2W"
+            # Get default owner
+            bus_i = transformer["f_bus"]
+            owner = bus_owner[bus_i]
+                        
+            # Get the source data
+            source_type = data["source_type"]
+
+            # Convert to two winding transformer
+            psse_comp = _pm2psse_2w_tran(transformer, owner, sbase, source_type)
+            
+            lines = [
+                "TRANSFORMER",
+                "TRANSFORMER TWO-WINDING LINE 1",
+                "TRANSFORMER TWO-WINDING LINE 2",
+                "TRANSFORMER TWO-WINDING LINE 3",
+                ]
+                
+        elseif type == "3W"
+            # Get default owner
+            bus_i = transformer["w1"]["f_bus"]
+            owner = bus_owner[bus_i]
+            
+            # Convert to Three winding transformer
+            psse_comp = _pm2psse_3w_tran(transformer, owner, sbase)
+            
+            lines = [
+                "TRANSFORMER",
+                "TRANSFORMER THREE-WINDING LINE 1",
+                "TRANSFORMER THREE-WINDING LINE 2",
+                "TRANSFORMER THREE-WINDING LINE 3",
+                "TRANSFORMER THREE-WINDING LINE 4",
+                ]
+        end
+        
+        for (psse_part, line) in zip(psse_comp, lines)
+            _print_pti_str(io, psse_comp, _pti_dtypes[line])
+        end
+    end
+
+    println(io, "0 / END OF TRANSFORMER DATA, BEGIN AREA DATA")
+
+    # Area Interchange
+    if haskey(data, "area interchange") 
+        for (_, area) in sort(data["area interchange"], by = (x) -> parse(Int64, x))
+            # Get Dict in a PSSE way and print it
+            psse_comp = _pm2psse_area_interchange(area)
+            _print_pti_str(io, psse_comp, _pti_dtypes["AREA INTERCHANGE"])
+        end
+    else
+        Memento.warn(_LOGGER, string("Skipping AREA INTERCHANGE data because it does not found"))
+    end
+    
+    println(io, "0 / END OF AREA DATA, BEGIN TWO-TERMINAL DC DATA")
+
+    # TODO : See how PM converts the DC line and do the oposite
+    Memento.warn(_LOGGER, string("Export TWO-TERMINAL DC data is not yet supported"))
+    
+    println(io, "0 / END OF TWO-TERMINAL DC DATA, BEGIN VOLTAGE SOURCE CONVERTER DATA")
+    
+    # TODO : See how PM converts the DC line and do the oposite
+    Memento.warn(_LOGGER, string("Export VSC data is not yet supported"))
```

Comment:
Can be removed, covered by the previous dcline warning.

Reply:
Done at the top of the function